### PR TITLE
Update GSMTC-Support-And-Popular-Apps.md

### PR DESCRIPTION
**Pull request by Cyberdroid1**
Trying out a text style. What do you say? Is it readable?

### CODE REVIEW

**Comment by ShankarBUS** [comment]
@Samuel12321, What do you say?

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (ShankarBUS) [inline docs/GSMTC-Support-And-Popular-Apps.md:31]: A big no for this. It would be easier to have the text combined. It's too distracting now

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (ShankarBUS) [inline docs/GSMTC-Support-And-Popular-Apps.md:28]: Bruh! Why? My statement was just eloborative and bold.

Why tho?

**Review by ShankarBUS — CHANGES_REQUESTED** [review]
Let's wait for @Samuel12321 approval.

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (Cyberdroid1) [reply docs/GSMTC-Support-And-Popular-Apps.md:28]: You see this big comment in table, you will see that it looks so ugly with a buttload of text in one of the cell in the table and nowhere else.

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (Cyberdroid1) [reply docs/GSMTC-Support-And-Popular-Apps.md:31]: Yeah, that's why I wanted to ask about this.

Comment (Cyberdroid1) [reply docs/GSMTC-Support-And-Popular-Apps.md:31]: Can we remove these distinctions all together? The audio only, video only, and both ones?

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (Cyberdroid1) [reply docs/GSMTC-Support-And-Popular-Apps.md:28]: What about adding a separate line to the doc for the same? It can contain this line and some more elaboration. It could be better

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (ShankarBUS) [reply docs/GSMTC-Support-And-Popular-Apps.md:31]: It would help segregate the clutter. So, let's keep them but not like that spacy like you changed.

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (ShankarBUS) [reply docs/GSMTC-Support-And-Popular-Apps.md:28]: Hmm, I agree on this one.

**Comment by Renzo904** [comment]
I know it would be probably a little pain, but what about moving on to a html table? IMHO it would be better bcs it has the `colspan` attribute so it would fix the Audio,video only issue

**Comment by ShankarBUS** [comment]
@Renzo904,

> I know it would be probably a little pain, but what about moving on to a html table? IMHO it would be better bcs it has the `colspan` attribute so it would fix the Audio,video only issue

Hmmm, @Cyberdroid1 could you try it?

**Comment by Renzo904** [comment]
> @Renzo904,
> 
> > I know it would be probably a little pain, but what about moving on to a html table? IMHO it would be better bcs it has the `colspan` attribute so it would fix the Audio,video only issue
> 
> Hmmm, @Cyberdroid1 could you try it?

Nvm, github doesnt accept html styles -_-, so it would be pretty much the same

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (Samuel12321) [reply docs/GSMTC-Support-And-Popular-Apps.md:28]: agreed

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (Cyberdroid1) [reply docs/GSMTC-Support-And-Popular-Apps.md:31]: We can use the h1, h2, h3 tags here in the table, right? How about using them?

**Comment by ShankarBUS** [comment]
@Cyberdroid1,

mi-mi

**Comment by Cyberdroid1** [comment]
What about now?

**Comment by Cyberdroid1** [comment]
Don't merge yet.

**Comment by ShankarBUS** [comment]
@Cyberdroid1,

I feel like the existing one was fine.

I do approve the truncation of the "Chromium and ...." and the "Play/Pause".
But I prefer to keep the distinction between media players. Because Audio players and Video players have a completely different set of supported features. So, let's keep it like that.

**Comment by Cyberdroid1** [comment]
As you wish, sire.

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (ShankarBUS) [inline docs/GSMTC-Support-And-Popular-Apps.md:32]: Bruh, why spaces only on some places? It's inconsistent. Could you remove this please?

**Review by ShankarBUS — CHANGES_REQUESTED** [review]

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (Cyberdroid1) [reply docs/GSMTC-Support-And-Popular-Apps.md:32]: The spaces dont appear in the doc.
Adding spaces in the raw looks good.

**Comment by Cyberdroid1** [comment]
How about now?

**Comment by ShankarBUS** [comment]
Now that's my boi!

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (Cyberdroid1) [reply docs/GSMTC-Support-And-Popular-Apps.md:32]: There are some big gaps after this, so did not do so for others.

**Comment by Cyberdroid1** [comment]
Merge then. Final from my side.

File: docs/GSMTC-Support-And-Popular-Apps.md
Comment (ShankarBUS) [reply docs/GSMTC-Support-And-Popular-Apps.md:32]: Hmmm. I know.

**Review by ShankarBUS — APPROVED** [review]

**Pull request merged**
